ping/tests/test_ping.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+import mock
+
+from datadog_checks.checks import AgentCheck
+from datadog_checks.ping import PingCheck
+from datadog_checks.errors import CheckException
+
+
+def mock_exec_ping():
+    return """FAKEPING 127.0.0.1 (127.0.0.1): 56 data bytes
+64 bytes from 127.0.0.1: icmp_seq=0 ttl=64 time=0.093 ms
+
+--- 127.0.0.1 ping statistics ---
+1 packets transmitted, 1 packets received, 0.0% packet loss
+round-trip min/avg/max/stddev = 0.093/0.093/0.093/0.000 ms"""
+
+
+def test_check(aggregator, instance):
```

Comment:
Is that possible to add an additional `integration` test also (aka actually run the ping command without mocking it)?

Reply:
Done, let me know what you think about it. Thanks!